relayer/application-standard.ts

```diff
@@ -55,6 +59,7 @@ export type StandardRelayerContext = LoggingContext &
 export class StandardRelayerApp<
```

Comment:
This class is starting to become a bit large s.t. if someone wants to change one of these defaults (e.g. not used missedVaa middleware) they need to copy a bunch of code. 

What about creating a `RedisRelayerApp` that is like the previous `RelayerApp` but configured with redis?

Reply:
I'd like to discuss a few things before doing that but I like it. Can we tackle that in a separate pr?

Reply:
Yeah agreed on separate pr 👍 
Happy to chat tomorrow